README.md

```diff
@@ -45,3 +55,17 @@ are welcome to [join](https://pyslackers.com/slack/).
 
 If you have a GitHub account, remember to star this repository so you can
 have quick access to this documentation. If not, bookmark this page.
+
+# Creating the Table of Contents
```

Comment:
Should this be in the `README.md` ? Maybe we should create a `CONTRIBUTING.md` :thinking: